src/workloads/setup.py

```diff
@@ -10,7 +10,7 @@
 
 
 # HISTORY.rst entry.
-VERSION = '0.1.0-preview'
```

Comment:
Since we already add `a1` as preview label, I think we should use `1.0.0a1` instead as `0.x.x` means preview as well.